packages/string-templates/src/index.ts

```diff
@@ -229,7 +230,10 @@ export function processStringSync(
     } else {
       return process(string)
     }
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
NAB: Is any needed?

Reply:
Sadly yes, without it there's a type error that fails the library tests.